thunder/tests/test_interpreter.py

```diff
@@ -274,14 +274,14 @@ def __bool__(self):
     bx = bool(x)
     jitting = True
     jbx = jit(bool)(x)
-    assert bx == jbx == False
+    assert bx == jbx is False
```

Comment:
[PEP 8](https://peps.python.org/pep-0008/#programming-recommendations) says comparing to truth values using `is` is even worse than using `==`. To my personal eyes, `is` looks totally fine though.